reflex-app/src/animations/CheckboxDemo/CheckboxDemo.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment, Component, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const checkStroke = keyframes`
+ 100% {
+   stroke-dashoffset: 0;
+ }
+`;
+
+const checkScale = keyframes`
+  100% {
+    transform: scale(.85);
+  }
+`;
+
+const checkHover = keyframes`
+  100% {
+    background-color: #7567F7;
+    border: 1px solid #7567F7;
+    stroke: #fff;
+  }
+`;
+
+const Checkbox = styled.div`
+  position: relative;
+  width: 36px;
+  height: 36px;
+
+  > input {
```

Comment:
What does the `>` do in SCSS?

Reply:
only applies to the immediate children of `input`